src/plugins/mission_raw/include/plugins/mission_raw/mission_raw.h

```diff
@@ -113,6 +128,26 @@ class MissionRaw : public PluginBase {
      */
     void download_mission_cancel();
 
+    /**
+     * @brief Uploads a vector of mission raw to the system (asynchronous).
+     *
+     * The mission raw are uploaded to a drone. Once uploaded the mission can be started and
+     * executed even if a connection is lost.
+     *
+     * @param mission_items Reference to vector of mission items.
+     * @param callback Callback to receive result of this request.
+     */
+    void upload_mission_async(const std::vector<std::shared_ptr<MissionRaw::MavlinkMissionItemInt>>& mission_raw,
+            result_callback_t callback);
+
+    /**
+     * @brief Cancel a mission upload (asynchronous).
+     *
+     * This cancels an ongoing mission upload. The mission upload will fail
+     * with the result `Result::CANCELLED`.
+     */
+    //TODO void upload_mission_cancel();
```

Comment:
Please either implement it or remove it.

Reply:
Not too lazy but kind in a rush with dev so I removed it :)